client/.eslintrc.json

```diff
@@ -0,0 +1,24 @@
+{
+    "env": {
+        "browser": true,
+        "commonjs": true,
+        "es2021": true
+    },
+    "extends": [
+        "eslint:recommended",
+        "plugin:react/recommended",
+        "plugin:@typescript-eslint/recommended"
```

Comment:
add "plugin:react/jsx-runtime"  to extend so I dont have to import react when writing jsx